experimental/dds/tree2/src/feature-libraries/editable-tree-2/README.md

```diff
@@ -46,12 +46,30 @@ There are four main usage modes for this API:
     This could be considered part of either reading API, but is being addressed separately here.
     APIs for receiving these changes take the form of registering callbacks for events.
     These events can be triggered by anything that impacts the content of the tree including local edits, remote edits, merges, rebasing local edits, transaction rollback, undo etc.
-    These can also be batched together, possibly eliminating intermediate states and reordering operations that don't impact eachother (like edits to different parts of the tree).
+    These can also be batched together, possibly eliminating intermediate states and reordering operations that don't impact each other (like edits to different parts of the tree).
     This means these event subscriptions should only be treated as changes to the content, and users of them should not attempt to ascribe meaning or providence to any particular event.
 
     Some control over these events is available via branching which provides snapshot isolation (among other things):
     this is done at a higher level than the tree API covered here but can be accessed via each tree entity's context property. (TODO: provide this access).
 
+To accommodate all of these at once in a single API surface, a few principals are followed in the API's design:
```

Comment:
```suggestion
To accommodate all of these at once in a single API surface, a few principles are followed in the API's design:
```
The design is heavily influenced by Yann, Nick, Taylor, Daniel and you (congrats), but I don't think that's what you meant.